### PR TITLE
Add free trial host value for tracks

### LINKED ISSUE #949: Free trial: set tracks host value to `ecommplan-freetrial`
p1676392842441739-slack-C9K5T0XSP

We need to set track host property to `ecommplan-freetrial`  in  `set_tracks_host_value` function to identify free trial stores for monitoring.

https://github.com/Automattic/wc-calypso-bridge/blob/a9e78a489ded5dcaea0a832143f0f25fd409c110/includes/class-wc-calypso-bridge-tracks.php#L80-L89

### Acceptance Criteria

- In a free trial store, track events are fired with prop `host =  ecommplan-freetrial`

Related https://github.com/Automattic/wc-calypso-bridge/pull/800 .

### PR DESCRIPTION
**Pull request by ilyasfoo**
### Changes proposed in this Pull Request:

Closes #949.

This PR adds sets tracks `host` prop to `ecommplan-freetrial` when it detects free trial plan.

### How to test the changes in this Pull Request:

1. On a free trial site, go to WooCommerce homescreen
2. Right click on anywhere on screen, view page source
3. Search for the text `properties.host`
4. Observe the value is set to `ecommplan-freetrial`.

### CODE REVIEW

**Review by chihsuan — APPROVED** [review]
Thanks for adding it! LGTM and tested well. 👍

File: readme.txt [hunk 1]
```diff
@@ -22,11 +22,6 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
-= Unreleased =
-* Use the theme color for completed task strikethrough #1006
-* Add free trial notice on WooCommerce Orders page #936
-* Change illustration in tasklist completed component #1007
-
```
Comment (chihsuan) [inline readme.txt:29]: Hey @ilyasfoo Don't we need to add changelog? Or we just add changelogs in release PR?

Comment (ilyasfoo) [reply readme.txt:29]: @chihsuan Doing this due to the coordination to release 2.0.5 in  p1677667881815459/1677664953.596819-slack-C03Q87XT1QF

Comment (ilyasfoo) [reply readme.txt:29]: Not the most ideal process tbh, but we haven't really discussed on how to improve it yet

Comment (chihsuan) [reply readme.txt:29]: Ah, Gotcha, thanks!

**Pull request merged**